c++/src/kj/compat/http.c++

```diff
@@ -2885,7 +2885,6 @@ public:
               // We must reset context on each message.
               decompressor.reset();
             }
-            
```

Comment:
Why the randomly-deleted blank lines?

Reply:
these lines consisted of spaces, so that probably threw something off. Not sure myself, but fixed.